upi/openstack/network.yaml

```diff
@@ -7,6 +7,8 @@
 
 - hosts: all
   gather_facts: no
+  vars:
+    _interfaces: ["{{ os_subnet }}"]
```

Comment:
how about we name this `router_interfaces` instead, so it becomes more descriptive?

Reply:
also can you try converting the string to list with `{{ os_subnet | list }}` instead?

Reply:
that would produce `['n', 'a', 'm', 'e']` i the os_subnet would hold `name` value.